packages/icons/icons.json

```
@@ -471,280 +471,280 @@
         "iconName": "cog",
-        "tags": "settings, circle,",
+        "tags": "settings, circle, , wheel, mechanical, tools, gear",
         "group": "interface",
         "codepoint": 61789
     },
     {
         "displayName": "Flag",
         "iconName": "flag",
-        "tags": "map, position, country, nationality",
+        "tags": "map, position, country, nationality, flagpole, flag pole",
         "group": "interface",
         "codepoint": 61870
     },
     {
         "displayName": "Pin",
         "iconName": "pin",
-        "tags": "map, position, safety pin, attach",
+        "tags": "map, position, safety pin, attach, thubtack, sharp, push pin, thumb tack",
         "group": "action",
         "codepoint": 62069
     },
     {
         "displayName": "Warning sign",
         "iconName": "warning-sign",
-        "tags": "notification, warning, triangle, exclamation mark, sign",
+        "tags": "notification, warning, triangle, exclamation mark, sign, alert, caution",
         "group": "interface",
         "codepoint": 62229
     },
     {
         "displayName": "Error",
         "iconName": "error",
-        "tags": "notification, failure, circle, exclamation mark, sign",
+        "tags": "notification, failure, circle, exclamation mark, sign, contour, warning, attention, look out",
         "group": "interface",
         "codepoint": 61851
     },
     {
         "displayName": "Info sign",
         "iconName": "info-sign",
-        "tags": "notification, information, circle, message, sign",
+        "tags": "notification, information, circle, message, sign, info, tooltip",
         "group": "interface",
         "codepoint": 61952
     },
     {
         "displayName": "Credit card",
         "iconName": "credit-card",
-        "tags": "payment, bank, transaction",
+        "tags": "payment, bank, transaction, webpage, document, \"i dont know",
         "group": "action",
         "codepoint": 61800
     },
     {
         "displayName": "Edit",
         "iconName": "edit",
-        "tags": "annotate, pen, modify",
+        "tags": "annotate, pen, modify, draw, signature",
         "group": "action",
         "codepoint": 61844
     },
     {
         "displayName": "History",
         "iconName": "history",
-        "tags": "past, reverse, circle, arrow",
+        "tags": "past, reverse, circle, arrow, out of date, refresh, clock, undo, rollback, rewind, revert, roll back, reverse time, counter",
         "group": "action",
         "codepoint": 61936
     },
     {
         "displayName": "Search",
         "iconName": "search",
-        "tags": "inspection, exploration, magnifying glass",
+        "tags": "inspection, exploration, magnifying glass, find, help",
         "group": "action",
         "codepoint": 62114
     },
     {
         "displayName": "Logout",
         "iconName": "log-out",
-        "tags": "arrow, leave",
+        "tags": "arrow, leave, next, share, open in new tab, exit",
```

we shouldn't conflate this with the real "share" icon

```suggestion
        "tags": "arrow, leave, next, exit, sign out, auth, authentication",
```